### ISSUE TITLE
Import update on file move not working for non-relative file paths

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/TypeScript#24544]
From https://github.com/Microsoft/vscode/issues/50707

**TypeScript Version:**  3.0.0-dev.20180530

**Search Terms:** 
- update paths on file move
- base url
- non-relative

**Code**
1. Clone this project: https://github.com/mjbvz/ts-update-paths-non-relative

The basic structure is:

```
tsconfig.json
sub/ 
    a.ts
    b.ts
```

`tsconfig.json`:

```json
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "es2016",
        "jsx": "preserve",
        "baseUrl": "."
    },
    "exclude": [
        "node_modules",
        "**/node_modules/*"
    ]
}
```

`a.ts`:

```ts
export const a = 123; 
```

`b.ts`

```ts
import { a } from 'sub/a';

export const b = 123;

console.log(a) 
```

1. Rename `sub/a.ts` to `sub/z.ts`

**Expected behavior:**
Prompt to update paths

**Actual behavior:**
No prompt. This works if you use relative paths

**Comment by andy-ms** [comment]
Looks like this was fixed by #24568. @mjbvz could you confirm?

**Comment by mjbvz** [comment]
Thanks. Looks fixed in 3.0.0-dev.20180602

This is back ported to 2.9.2 already as well?

**Comment by andy-ms** [comment]
Yep, #24568 is in the `release-2.9` branch and #24305 is the `master` version.
